src/components/ExpandableContainer/ExpandableContainer.style.tsx

```diff
@@ -19,11 +19,11 @@ export const ExpandButton = styled(LargeOutlinedButton)<{
 `;
 
 export const InnerContent = styled.div<{
-  $collapsedHeight: number;
+  $collapsedHeight: string;
   $collapsed: boolean;
 }>`
   height: ${({ $collapsed, $collapsedHeight }) =>
-    $collapsed ? `${$collapsedHeight}px` : 'fit-content'};
+    $collapsed ? `${$collapsedHeight}` : 'fit-content'};
```

Comment:
tiny thing -- no need to make $collapsedHeight a template literal. can nix the backticks and just declare `$collapsedHeight` (like you declare `$collapsed`)